.pre-commit-hooks.yaml

```diff
@@ -28,3 +28,10 @@
   entry: sort-package-xml
   language: python
   files: package.xml$
+
+- id: ros-include-guard
+  name: check include guard
```

Comment:
To make the style consistent.

```suggestion
  name: Check include guard
```

Reply:
How about unifying to lowercase?
```
check json
check for merge conflicts
check toml
check xml
check yaml
detect private key
fix end of files
mixed line ending
trim trailing whitespace
markdownlint
prettier
yamllint
Prettier package.xml
Sort package.xml
shellcheck
shfmt
isort
black
flake8
clang-format
cpplint
```

Reply:
It's fine, but I feel making them the same as their IDs is better because people can just copy-and-paste the IDs.

Also, IMO we can drop the `description` fields because we write READMEs instead.

References:
- https://pre-commit.com/#creating-new-hooks
- https://github.com/pre-commit/mirrors-prettier/blob/ea782651a7e32f40a3d13b76c79d5a2474ee8723/.pre-commit-hooks.yaml

@KeisukeShima What do you think?

Reply:
My personal opinion is that lowercase is better for consistency with other tools.

Reply:
Thank you for your comment. What are `other tools`?

Reply:
Other tools are things like `prettier`, `shfmt`, etc. It is a good idea to use the same style as those.

Reply:
It seems the format in the standard hooks repo has changed recently.
https://github.com/pre-commit/pre-commit-hooks/blob/v4.0.1/.pre-commit-hooks.yaml
https://github.com/pre-commit/pre-commit-hooks/blob/v4.1.0/.pre-commit-hooks.yaml

Let's follow this decision!

Reply:
Hmm, it seems there are no big reasons but due to the preference of the original author.
https://github.com/pre-commit/pre-commit-hooks/pull/670#issuecomment-945719372
https://github.com/pre-commit/pre-commit-hooks/pull/413#issuecomment-534513444

I'm okay with using either style of `check-json` or `check json`.

Considering that it will be displayed as the following, in order to be the same as `pre-commit/pre-commit-hooks`, `check json` is better?
![image](https://user-images.githubusercontent.com/31987104/155089048-07bf262d-d281-4e0f-a3f2-ab14640aac10.png)

Reply:
Fixed. https://github.com/tier4/pre-commit-hooks-ros/pull/32/commits/1074ab2c929fc3b5c076255eb63cb730ea8eace1